SETUP/UNICODE.md

```diff
@@ -0,0 +1,102 @@
+# Unicode support
+
+The DP codebase fully supports UTF-8, both in the HTML rendered to the user
+as well as in page texts.
+
+Rather than open up the floodgates and allow proofreaders to input any of
+the millions of Unicode character into the proofreading interface, we allow
+project managers to restrict what Unicode characters are valid for a project.
+This ensures that proofreaders don't insert characters that don't rightfully
+belong in the project but look similar to those that do (Kappa for a 'K'
+as an example).
+
+Both the UI and the server-side normalization enforce that only supported
+characters are used. This allows the code to support the full set of Unicode
+characters but reduces the possibility of invalid characters in a given project.
```

Comment:
Should there be some mention here of the fact that this enforcement applies only to the text in the projects, and that project title and author are not held to the same restrictions? (Even once we force normalization on title and author, we're not going to force them to be held to the same charsuites that the project text is, are we?)

Reply:
Good idea. I've added more details as well as info around the phpBB limitations.